### ISSUE TITLE
[Bug] Unordered List symbol vanishes in Edit Mode

### ISSUE THREAD

**Issue by Mondrethos** [SlRvb/Obsidian--ITS-Theme#286]
### Describe the bug

Type - to initiate unordered list, the unordered list symbol vanishes even though the text still gets intended

**Expected behavior**

Some sort of indicator that a list is being used. This issue was fixed for numbered lists here: https://github.com/SlRvb/Obsidian--ITS-Theme/issues/242

### To Reproduce

Type - to initiate an unordered list, the unordered list symbol vanishes, even though the text still gets indented.

### OS
Linux

- [X] Desktop
- [ ] Mobile

### Versions
- Obsidian Version: 1.5.8
- ITS Theme Version: 1.2.45

**Comment by SlRvb** [comment]
Do you happen to have any plugins toggled on or any potential stuff toggled on in the style settings that might cause this to happen? I can't see to recreate this personally to see what's happening 🤔

**Comment by Mondrethos** [comment]
I've disabled every plugin, and it only comes back when I disable the Style Settings plugin, which I use to turn ITS blue instead of the default red.

**Comment by SlRvb** [comment]
Hmmm, any chance I could get an export of your style settings here?

**Comment by Mondrethos** [comment]
I apologize for the delay. @Sl

Here's what I get when I click export under Style Settings:

```
{
  "ITS@@slrvb-b": false,
  "ITS@@slrvb-g": false,
  "ITS@@drwn": false,
  "ITS@@nord": false,
  "ITS@@notion": false,
  "ITS@@its-d": false,
  "ITS@@select": "none",
  "ITS@@s-d": true,
  "ITS@@list-bullet-side-lines": true,
  "ITS@@alternate-indent": false,
  "ITS@@indentation-guide-color@@dark": "#2182D7"
}
```

**Comment by SlRvb** [comment]
This should be fixed _now_ on ITS v1.2.85!
